src/pages/pass-finalisation/pass-finalisation.ts

```diff
@@ -229,4 +239,20 @@ export class PassFinalisationPage extends PracticeableBasePageComponent {
     return subscription;
   }
 
+  d255Changed(d255: boolean): void {
```

Comment:
same as above - just double check if it supposed to be here since it's not on the original cherry-pick